### PR TITLE
Refactor/language select update

### PR DESCRIPTION
**Pull request by mexi-cano**
What (if any) features are you implementing?
- Selected language is always shown. If none, 'Choose Language' will show. Select chevron now appear.

What (if anything) did you refactor?
- Replaced globe icon. 

Screenshot 2023-12-04 at 11 03 05 AM

Screenshot 2023-12-04 at 11 17 18 AM

Screenshot 2023-12-04 at 11 02 53 AM

### CODE REVIEW

File: src/Components/Header/Header.js [hunk 1]
```diff
@@ -89,9 +86,7 @@ const Header = ({ handleTextfieldChange }) => {
               open={isLanguageSelectOpen}
               onOpen={handleOpenLanguage}
               onClose={handleCloseLanguage}
-              IconComponent={LanguageIcon}
-              renderValue={() => setRenderValue()}
-              sx={{ '& .MuiSvgIcon-root': { right: '1.5rem', color: '#FFFFFF' } }}
+              sx={{ '& .MuiSvgIcon-root': { color: '#FFFFFF' } }}
```
Comment (CalebPena) [inline src/Components/Header/Header.js:89]: Out of curiosity, do we need this line?

**Review by CalebPena — APPROVED** [review]
Looks good, just one quick question.

**Review by Mdemenko1 — COMMENTED** [review]
@CalebPena Does @mexi-cano needs to change base branch?

**Comment by CalebPena** [comment]
> @CalebPena Does @mexi-cano needs to change base branch?

Yes, please change the base branch to the branding changes feature branch.

File: src/Components/Header/Header.js (hunk 1 above)
Comment (mexi-cano) [reply src/Components/Header/Header.js:89]: Yes, this line makes the Select chevron icon white.

**Review by sydney-devine — APPROVED** [review]
Visuals look good on my end.

**Pull request merged**